ci/build_docs.sh

```diff
@@ -23,14 +26,11 @@ PYTHON_CHANNEL=$(rapids-download-conda-from-s3 python)
 rapids-mamba-retry install \
   --channel "${CPP_CHANNEL}" \
   --channel "${PYTHON_CHANNEL}" \
-  libraft \
-  libraft-headers \
-  pylibraft \
-  raft-dask
+  "libraft=${RAPIDS_VERSION_MAJOR_MINOR}" \
+  "libraft-headers=${RAPIDS_VERSION_MAJOR_MINOR}" \
+  "pylibraft=${RAPIDS_VERSION_MAJOR_MINOR}" \
+  "raft-dask=${RAPIDS_VERSION_MAJOR_MINOR}"
```

Comment:
Would this be better suited for a DFG file key?

Reply:
That'd be a more invasive and therefore time-consuming change.

Across these PRs, I'm intentionally not involving `dependencies.yaml` in this because that introduces other complexity I don't want to invest the time in taking on right now. https://github.com/rapidsai/build-planning/issues/22 tracks the work of doing that.